.eslintrc.json

```diff
@@ -3,7 +3,15 @@
         "browser": true,
         "es6": true
     },
-    "extends": "eslint:recommended",
+    "extends": [
+      "eslint:recommended",
+      "plugin:react/recommended"
```

Comment:
This is to make ESLint understand React specifics. We can configure it further later (see [doc](https://github.com/yannickcr/eslint-plugin-react)).